src/Chapter03/Chapter03.csproj

```
@@ -26,7 +26,13 @@
     <Compile Update="Listing03.02.DereferencingAnUnassignedVariable.cs">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Compile>
-    <Compile Update="Table03.03.CommonArrayCodingErrors.cs">
+    <Compile Update="Table03 - Copy.03.b.CommonArrayCodingErrors.cs">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Compile>
+    <Compile Update="Table03.03.a.CommonArrayCodingErrors.cs">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Compile>
+    <Compile Update="Table03.03.b.CommonArrayCodingErrors.cs">
```

This is only copying .a and .b. Should this include all of them? Though given that it appears the tests are passing perhaps we don't need to copy any of them?
```suggestion
    <Compile Update="Table03.03.*.CommonArrayCodingErrors.cs">
```